test/api_users_test.go

```diff
@@ -137,7 +136,7 @@ func Test_client_UsersAPIService(t *testing.T) {
 		updateUser.Name = &newName
 		updateUser.RootRole = &rootRole
 
-		resp, httpRes, err := apiClient.UsersAPI.UpdateUser(context.Background(), id).UpdateUserSchema(*updateUser).Execute()
+		resp, httpRes, err := apiClient.UsersAPI.UpdateUser(context.Background(), user.Id).UpdateUserSchema(*updateUser).Execute()
```

Comment:
Caused by a change in upstream types. This becomes a string internally anyway so it won't break against Unleash but does require users of this project to change their types when using the API